presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogsHandler.java

```diff
@@ -56,13 +63,36 @@
     private static final String DATE_GROUP = "date";
     private static final String LEVEL_GROUP = "level";
     private static final Pattern LOG_REGEX = Pattern.compile(
-            "^(?<" + DATE_GROUP + ">[0-9]{4}-[0-9]{2}-[0-9]{2}T" +
+            "^(?<date>[0-9]{4}-[0-9]{2}-[0-9]{2}T" +
                     "[0-9]{2}:[0-9]{2}:[0-9]{2}(\\.[0-9]{3})?[+-][0-9]{4})" +
-                    "\t(?<" + LEVEL_GROUP + ">[A-Z]+)\t(?<thread>[^\t]+)" +
+                    "\t(?<level>[A-Z]+)\t(?<thread>[^\t]+)" +
                     "\t(?<class>[^\t]+)\t(?<message>.*)$");
+    private static final String DEFAULT_ENTRY = "0000-01-01T00:00:00.000+0000" +
+            "\tALL\t[none]\t[none]\tThis log entry was not preceded by a header:";
 
     private LogsHandler() {}
 
+    public static Response getLogList()
+    {
+        String fileList;
+        try (Stream<java.nio.file.Path> files = Files.list(LOG_DIRECTORY)) {
+            fileList = files.filter(Files::isRegularFile)
```

Comment:
We don't get to name Presto's log files.
There shouldn't be any harm in this; the log directory doesn't tent to contain files that it shouldn't.